magma_access_gateway_configurator/__init__.py

```diff
@@ -64,6 +76,51 @@ def validate_args(args):
         raise FileNotFoundError(f"Given Root CA PEM path {args.root_ca_path} doesn't exist!")
 
 
+def load_public_key_to_base64der(key_file):
```

Comment:
Please add typing for `key_file` as well as return typing

Reply:
Done.